src/main/java/io/stargate/sgv2/jsonapi/service/schema/tables/ApiVectorIndex.java

```diff
@@ -46,6 +46,11 @@ private ApiVectorIndex(
     this.sourceModel = sourceModel;
   }
 
+  @Override
+  public ApiIndexType indexType() {
+    return ApiIndexType.VECTOR;
+  }
+
```

Comment:
Add override indexType() method for `ApiVectorIndex` class. Otherwise, the index type `ApiIndexType.REGULAR` will be inherent from the parent class `ApiSupportedIndex`